src/api/goods.js

```diff
@@ -6,6 +6,20 @@ export function getAll() {
     .then(response => response.json());
 }
 
-export const get5First = () => {};
+export const get5First = () => (
+  getAll()
+    .then(result => (
+      result
+        .sort(({ name: first }, { name: scnd }) => (
```

Comment:
try not to use shortened words for variables` names
```suggestion
        .sort(({ name: first }, { name: second }) => (
```